lib/hook-types/slack.js

```diff
@@ -10,18 +10,37 @@ function test(hook_url) {
   return false;
 }
 
-function fire(id, hook_url, type, incoming, hmac, headers = {}, token = null) {
-  if (!test(hook_url)) {
-    throw new Error('unable to fire webhook:', hook_url, 'is not a valid slack endpoint');
-  }
+function formatter(incoming) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function 'formatter' has a complexity of 11. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5661986)